tests/installation/select_system_role.pm

```diff
@@ -0,0 +1,51 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test module to handle System Role during installation:
+#          - Validate default system role selection (optional)
```

Comment:
I wonder if we want to have separate module to validate default and module which just selects system role, as we don't validate default in each case.
I would also mention that this module uses REST API, as we have tests/installation/select_system_role.pm and tests/installation/system_role.pm which do same thing. I will address this in `accept_license.pm` module I've recently added.

Reply:
good point, the granularity convinced me here, as far was we have some folder with all those little 'steps' look nicer.